Makefile

```diff
@@ -7,7 +7,7 @@ test:
 
 install:
 	$(eval GO_LINKER_VALUE := $(shell git describe --tags --always | sed s/^v//))
-	go install -a -ldflags "-X ${GO_LINKER_SYMBOL} ${GO_LINKER_VALUE}" ./...
+	go install -a -ldflags "-X ${GO_LINKER_SYMBOL}=${GO_LINKER_VALUE}" ./...
```

Comment:
This sets the latest tag to a piece of metadata for the binary?

Reply:
Sorry for the delayed response, yes you have it exactly!